### DISCUSSION TITLE
fields:  - "message" terms_window_size:   days: 90

### DISCUSSION THREAD

**Discussion by commonReply** [jertel/elastalert2#868] [Q&A] [answered]
How can this configuration ensure that the matching content will not be repeated? I started this configuration, but there are still duplicate logs in the index ，The configuration is as follows。

name: test

type: new_term
index: test-*

fields:
 - "message"
terms_window_size:
  days: 90

num_events: 1

timeframe:
  minutes: 1
realert:
  minutes: 0

**Comment by nsano-rururu** [comment, marked as answer]
Please read https://github.com/jertel/elastalert2/discussions/11 and describe your environment, versions of software being used, and then format your message as instructed so that we can read the rule with all included indentation.
